examples/deadline/All-In-AWS-Infrastructure-Basic/scripts/configure_worker.sh

```diff
@@ -0,0 +1 @@
+mkdir test_dir 
```

Comment:
This is missing the:

```bash
#!/bin/sh
```

at the top.

Also, a comment would help to document that this script is only here to illustrate how to add custom user data when using our `WorkerInstanceFleet` and `WorkerInstanceConfiguration` constructs